pytest_tests/lib/helpers/http_gate.py

```diff
@@ -244,10 +241,11 @@ def upload_via_http_gate_curl(
     try:
         response_json = json.loads(output)
     except json.JSONDecodeError:
-        raise AssertionError(f'Invalid JSON response: {output}')
-    if 'object_id' not in response_json:
+        raise AssertionError(f"Invalid JSON response: {output}")
+    if "object_id" not in response_json:
```

Comment:
Why these changes?

Reply:
it is from black formatter, all files I ran through it